kubernetes/client/configuration.py

```diff
@@ -75,7 +75,7 @@ class Configuration(object):
     def __init__(self, host="http://localhost",
                  api_key=None, api_key_prefix=None,
                  username=None, password=None,
-                 discard_unknown_keys=False,
+                 proxy=None, discard_unknown_keys=False,
```

Comment:
Sorry for the delay. Unfortunately I think this file is generated, any changes here would be overwritten on a version bump. In order to make permanent changes to this file the PR would need to go to the generator repo, then we'd need to generate from the new version.

Reply:
oh I see, that makes sense. Thank you for the information